distribution/lib/Standard/Base/0.0.0-dev/src/Internal/Array_Like_Helpers.enso

```diff
@@ -29,6 +29,9 @@ from project.Data.Index_Sub_Range import drop_helper, Index_Sub_Range, take_help
 from project.Data.Ordering import Comparable
 from project.Data.Range.Extensions import all
 
+# often used in tests
```

Comment:
As discussed some time ago, can we put all the `polyglot java import`s in some special file (maybe something like `test/Base_Tests/src/Polyglot_Imports.enso`?) that will contain all these imports that are necessary only to make the native-image work? The [EnsoLibraryFeature](https://github.com/enso-org/enso/blob/16c1b74218f6ca15f7a2d9dc96356d67c39e499a/engine/runner/src/main/java/org/enso/runner/EnsoLibraryFeature.java) can pick that file and register all the mentioned classes for reflection.

Reply:
Why should that be a special file? What is wrong about discovering the imports from various places in the project?

Reply:
Because the imports introduced in this PR, along with others introduced with the [previous NI-related PR](https://github.com/enso-org/enso/pull/9997) are here only to fix running of `test/Base_Tests` with the NI generated engine-runner. Running `test/Base_Tests` with the standard JVM engine-runner works even without these imports. This makes these imports special. Therefore, I would rather see them in a separate file with a comment explaining that these imports are only necessary to make the `EnsoLibraryFeature` work.

Reply:
That is a very good idea! I have misunderstood you - I thought you wanted to move all kinds of Java imports into a single file - which seemed strange.

But I understand now that you wanted to move only 'optional' imports there, right? (i.e. imports not needed for `Base` but needed for tests)

Reply:
Yes, special file was the solution in https://github.com/enso-org/enso/pull/9997/commits/aaa5e9ec9ec39b510e977cfa67baa7f649b0d2c5

Reply:
Fixed in dc383710b5b901808d53a8a5c0b23317b67f718e